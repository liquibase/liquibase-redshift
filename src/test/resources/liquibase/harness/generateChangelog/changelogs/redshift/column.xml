<?xml version="1.1" encoding="UTF-8" standalone="no"?>
<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-latest.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
    <changeSet author="test" id="1">
        <createTable tableName="test_table">
            <column name="test_column" type="INTEGER"/>
            <column name="varcharColumn" type="VARCHAR(25)"/>
            <column name="intColumn" type="INTEGER"/>
            <column name="dateColumn" type="date"/>
        </createTable>
    </changeSet>
</databaseChangeLog>