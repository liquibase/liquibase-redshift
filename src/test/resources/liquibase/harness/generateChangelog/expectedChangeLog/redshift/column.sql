CREATE TABLE "test_table" ("test_column" INTEGER, "varcharColumn" VARCHAR(25), "intColumn" INTEGER, "dateColumn" date);
