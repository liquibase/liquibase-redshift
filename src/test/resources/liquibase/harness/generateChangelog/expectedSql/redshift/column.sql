CREATE TABLE "test_table" ("test_column" INTEGER, "varcharcolumn" VARCHAR(25), "intcolumn" INTEGER, "datecolumn" date);