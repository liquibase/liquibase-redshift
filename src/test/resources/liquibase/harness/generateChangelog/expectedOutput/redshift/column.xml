<?xml version="1.0" encoding="UTF-8"?>
<databaseChangeLog
        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
    <changeSet id="1" author="as">
        <createTable tableName="test_table">
            <column name="test_column" type="INTEGER"/>
            <column name="varcharcolumn" type="VARCHAR(25)"/>
            <column name="intcolumn" type="INTEGER"/>
            <column name="datecolumn" type="date"/>
        </createTable>
        <rollback>
            <dropTable tableName="test_table"/>
        </rollback>
    </changeSet>
</databaseChangeLog>